force-app/main/default/lwc/participantAdder/participantAdder.js

```diff
@@ -75,29 +75,35 @@ export default class ParticipantAdder extends LightningElement {
         if (result.data) {
             this.serviceScheduleModel = JSON.parse(JSON.stringify(result.data));
             this.extractLabels(this.serviceScheduleModel.labels.serviceParticipant);
-            this.isLoaded = true;
         } else if (result.error) {
             console.log(JSON.stringify(result.error));
         }
     }
 
-    extractLabels(labels) {
-        if (!labels) {
+    @wire(getExistingParticipantContactIds, { scheduleId: "$recordId" })
+    wiredExistingParticipants(result) {
+        if (!(result.data || result.error)) {
             return;
         }
 
-        Object.keys(labels).forEach(label => {
-            let value = labels[label];
-            this.labels[label] = value;
-        });
+        if (result.data) {
+            this.existingParticipants = result; //cache for refreshing
+        } else {
+            console.log(JSON.stringify(result.error));
```

Comment:
Is leaving this console log in source intentional?

Reply:
yes we are currently using console to log errors because we don't want to throw a toast unless there is user interaction. I believe we have a work item to design an on screen error so we can display these errors in the component itself vs throwing a toast but for now we are logging them so at least we can troubleshoot a bit easier.